code/controllers/subsystem/vote.dm

```diff
@@ -188,14 +188,14 @@ SUBSYSTEM_DEF(vote)
 				for(var/valid_map in maps)
 					choices.Add(valid_map)
 			if("custom")
-				question = stripped_input(usr,"What is the vote for?")
+				question = tgui_input_text(usr, "What is the vote for?", "Custom Vote", max_length = MAX_MESSAGE_LEN)
```

Comment:
does this have the exact same stripping semantics as stripped_input?

Reply:
going to html_encode all tgui text input by default, but you can simply ```encode = FALSE``` this behavior. so to answer your question, yes, it does now